### DISCUSSION TITLE
Clientseitige Marker (Koordinaten, Stationierung)

### DISCUSSION THREAD

**Discussion by cmairamhof** [e-netze/webgis-community#52] [General]
sollen auch wieder selektiv aus der Karte gelöscht werden können.

Der Wunsch, dass die Stationierung auch im Kartenbild (nicht nur im Ausdruck) beschriftet werden kann ist bei den Anwendern schon sehr groß.

![image](https://user-images.githubusercontent.com/79247887/230003084-a2a84b1e-0d0b-4cd8-bb54-d76c3252f845.png)

**Comment by gis-eni** [comment]
Vielleicht hilft ein update

**Comment by gis-eni** [comment]
Möglich wäre die Stationierungswerte im Werkzeugdialog anzuzeigen, ähnlich Koordinaten Tool. Marker mit Nummern

**Comment by gis-eni** [comment]
Die Stationierungswerte werden ab Build 5.23.1601 im Werkzeug Dialog angezeigt. Die Marker haben entsprechende Nummern für die Zuordnung:

![image](https://user-images.githubusercontent.com/62870568/233352117-84348b10-3750-4bb6-901d-c70235eddadf.png)
